main.go

```diff
@@ -130,7 +130,7 @@ func main() {
 			}
 
 			fmt.Println("Initializing chain service and connecting to " + chainUrl + "...")
-			chainService, err := chainservice.NewEthChainService(chainUrl, chainPk, common.HexToAddress(naAddress), common.Address{}, common.Address{}, os.Stdout)
```

Comment:
Why not let's go one step further and actually deploy and pass these addresses in?

I know we passed nil values in previously, but I'm uneasy about baking in these dummy values to our prod code (could be done in test code only).

Reply:
Updated in https://github.com/statechannels/go-nitro/pull/1293/commits/a936845689c590be73fdd619a2c5f307c690f54a

Reply:
Thanks! Will we need to update our rpc test orchestration script now?

Reply:
That was done in https://github.com/statechannels/go-nitro/commit/a936845689c590be73fdd619a2c5f307c690f54a too! The `start-rpc-servers` script has been updated to pass in the required addresses

Reply:
Of course it was 🤦